src/firmware_new/boards/frankie_v1/io/drivetrain_unit.h

```diff
@@ -1,21 +1,42 @@
 #pragma once
 
+#include "firmware/app/control/wheel_controller.h"
 #include "firmware_new/boards/frankie_v1/io/allegro_a3931_motor_driver.h"
 
 typedef struct DrivetrainUnit DrivetrainUnit_t;
 
 /**
  * Create a drivetrain unit
+ *
  * @param [in] motor_driver The driver attached to the motor in this drivetrain unit
+ *
+ * @param [in] controller The wheel controller attached to this drivetrain unit
+ *
  * @return The created DrivetrainUnit
  */
-DrivetrainUnit_t* io_drivetrain_unit_create(AllegroA3931MotorDriver_t* motor_driver);
+DrivetrainUnit_t* io_drivetrain_unit_create(AllegroA3931MotorDriver_t* motor_driver,
+                                            WheelController_t* controller);
 
 /**
  * Apply the given force to the given DrivetrainUnit
  *
  * @param [in] drive_train_unit The DrivetrainUnit to apply force to
- * @param force_newtons The force to apply, in newtons.
+ *
+ * @param new_speed_command [in] The newly requested speed in rad/s. Positive values imply
+ * CLOCKWISE rotation and negative CCW.
```

Comment:
should specify rotation relative to something. IIRC we have a "looking along the motor shaft" wording somewhere that we should use for consistency.